service.subtitles.legendastv/resources/lib/LTVutilities.py

```diff
@@ -0,0 +1,257 @@
+# -*- coding: utf-8 -*-
+
+import os
+import re
+import sys
+import xbmc
+import xbmcvfs
+import shutil
+import unicodedata
+import urllib, urllib2
+import ssl
+
+try: import simplejson as json
```

Comment:
please drop, simplejson is no longer needed as of Leia

Reply:
You mean I should just `import json`?

Reply:
Yes, use the builtin Json module. It used to have problems in helix (python2.6) that being the reason for simplejson as a fallback. No longer needed by today